example/sample_data/sample_user.csv

```diff
@@ -1,4 +1,4 @@
 email,first_name,last_name,full_name,github_username,team_name,employee_type,manager_email,slack_id,role_name
-roald.amundsen@example.org,Roald,Amundsen,"Roald Amundsen",lyft,"Team Amundsen",sailor,"phboss@example.org",ramundzn,swe
+roald.amundsen@example.org,Roald,Amundsen,"Roald Amundsen",amundsen,"Team Amundsen",sailor,"phboss@example.org",ramundzn,swe
```

Comment:
Nit, it builds a link on the Octocat icon in the user profile page so should probably be pointing to https://github.com/amundsen-io and not this dude https://github.com/amundsen (who I guess will get more Github notifications in the future 🙂)
```suggestion
roald.amundsen@example.org,Roald,Amundsen,"Roald Amundsen",amundsen-io,"Team Amundsen",sailor,"phboss@example.org",ramundzn,swe
```